client/core/bot.go

```diff
@@ -1246,27 +1247,29 @@ func (c *Core) feeEstimates(form *TradeForm) (swapFees, redeemFees uint64, err e
 			if err2 != nil {
 				return 0, 0, fmt.Errorf("error getting swap estimate (%v) and single-lot estimate (%v)", err, err2)
 			}
+			err = nil
 		} else {
 			return 0, 0, fmt.Errorf("error getting swap estimate: %w", err)
 		}
 	}
 
 	preRedeemForm := &asset.PreRedeemForm{
+		Version:         toWallet.version,
 		Lots:            lots,
 		FeeSuggestion:   redeemFeeSuggestion,
 		SelectedOptions: form.Options,
-		AssetConfig:     toAsset,
 	}
 	redeemEstimate, err := toWallet.PreRedeem(preRedeemForm)
 	if err == nil {
 		redeemFees = redeemEstimate.Estimate.RealisticWorstCase
 	} else {
-		if bw, is := fromWallet.Wallet.(asset.BotWallet); is {
+		if bw, is := toWallet.Wallet.(asset.BotWallet); is {
```

Comment:
Pretty sure the `fromWallet` part was copy pasta from the `PreSwap`->`SingleLotSwapFees` fallback pattern above.